### ISSUE TITLE
Ignite Problems

### ISSUE THREAD

**Issue by TRYAGAIN21** [TheMysticSword/Skillsmas#18]
The Lit Nano-Rocket's ignite damage appears to always be 1, no matter what. (Didn't do too much testing, but ignite from Flame Bolt certainly deals more damage even at level 1.)

**Comment by TheMysticSword** [comment]
Ignite's per-tick damage deals either 10% of your damage, or 1% of the target's max HP, whichever is the lowest. So you're very likely to see 1-damage numbers on the first few stages. Nothing much I can do, that's just how base game burn works.

And yeah, Revitalizing healing *is* multiplied by proc coefficient.

**Comment by TRYAGAIN21** [comment]
Ah, I see. That explains a lot. For whatever reason, I realize, during what little testing I did do, ignite was doing 7-8 damage _per tick_ at level _1_...? This has not reoccurred.

**Comment by TheMysticSword** [comment]
> Correct me if I'm wrong, but Stronger Burn works like regular burn, but with increased damage per tick, right?

Yep. But I can't really use that, or else my mod would need the player to own the DLC and have it enabled.

Anyway, I don't really have any intentions to change how the skill interacts with a base game mechanic, for the sake of consistency. I'd rather leave it up to someone else to eventually make a mod that somehow makes Ignite an overall more interesting debuff (I'm *really* not a fan of what they did to it in the DLC update lol)

**Comment by TRYAGAIN21** [comment]
Oh, right. I forgot it was a DLC item. Well, thank you for your time.
